docs/mxdoc.py

```diff
@@ -270,6 +270,8 @@ def _get_jupyter_notebook(lang, lines):
     for in_code, blk_lang, lines in _get_blocks(lines):
         if blk_lang != lang:
             in_code = False
+        # Exclude lines containing <!--no-notebook-->
+        lines = [line for line in lines if "<!--no-notebook-->" not in line]
```

Comment:
how about something like `<!--hide-line-->` or `<!--skip-render-->`?

Reply:
Can the skip happen outside of the for loop?

Reply:
I feel it is better to be specific about notebooks, since it is skipping rendering only for the jupyter notebooks. The line is rendered in html and markdown, the `<!--no-notebook-->` part being treated as comment in html and markdown won't appear at all. However in the notebooks, the entire line is skipped.

Reply:
As for skipping outside of the for loop, I am not sure what you mean? In this current implementation we will only go once through each line. We could do it outside in a `_filter_lines(lines)` function for example, but that wouldn't improve the efficiency which is already linear.

Reply:
how about `<!--skip-notebook-->` ?

Reply:
If you do the skipping outside of for loop, you give less lines to `_get_blocks`
Maybe `<!--notebook-skip-line-->`? The hope is that the comment is clear without ever going to the README.

Reply:
:+1: changed, will add tutorial once it has completed review